### PR TITLE
Ac404 30 displaying terminals

### PR DESCRIPTION
**Pull request by mkljaic20**
Added screens for displaying merchants and deleting them. Created classes that connect to the backend and fetch the data.

### CODE REVIEW

**Review by jivancic20 — DISMISSED** [review]
Looks good, nice job

File: app/src/main/java/com/Found404/paypro/ui/pages/AddingMerchantsPage.kt [hunk 1]
```diff
@@ -42,4 +86,86 @@ fun AddingMerchants(
...
+            onConfirm = {
+                onDeleteTerminal(selectedTerminalId)
+                showPopup = false
+            },
+            onCancel = { showPopup = false }
+        )
+    }
+}
+@Composable
+fun DeleteTerminalPopup(terminalId: String, onConfirm: () -> Unit, onCancel: () -> Unit) {
```
Comment (timjuic) [inline app/src/main/java/com/Found404/paypro/ui/pages/AddingMerchantsPage.kt]: Please place this composable in separate file in order to keep this page cleaner. You put it in `components` and call it `DeleteTerminalModal` or `DeleteTerminalPopup`

File: app/src/main/java/com/Found404/paypro/ui/pages/AddingMerchantsPage.kt [hunk 2]
```diff
@@ -42,4 +86,86 @@ fun AddingMerchants(
             Icon(imageVector = Icons.Default.Add, contentDescription = "Add")
         }
     }
-}
\ No newline at end of file
+}
+
+@Composable
+fun MerchantItem(merchant: MerchantResponse, onDeleteTerminal: (String) -> Unit) {
```
Comment (timjuic) [inline app/src/main/java/com/Found404/paypro/ui/pages/AddingMerchantsPage.kt]: MerchantItem can be moved to a separate component file.

**Review by timjuic — CHANGES_REQUESTED** [review]
Great job on the code. I only advise you to move mentioned functions to a separate files for better organization and readability.

Also it seems that this doesnt include the most recent code from develop, so there might be conflicts when merging

**Review by dominikn1 — DISMISSED** [review]
Looks good!

**Review by timjuic — APPROVED** [review]
Looks good now. Good job

**Review by dominikn1 — APPROVED** [review]
Changes look on point.

**Pull request merged**